src/core/chronology.jl

```diff
@@ -0,0 +1,45 @@
+struct Consecutive <: AbstractChronology end
+
+struct Synchronize <: AbstractChronology
+    from_steps::Int64    #number of time periods to grab data from
+    to_executions::Int64 #number of times to run using the same data
+    function Synchronize(;from_steps, to_executions)
+        new(from_steps, to_executions)
+    end
+end
+
+struct RecedingHorizon <: AbstractChronology
+    step::Int64
+    function RecedingHorizon(;from_step::Int64=1)
+        new(step)
+    end
+end
+
+function check_chronology(synch::Synchronize,
```

Comment:
sometimes synchronized gets shortened to "synch" and sometimes it gets shortened to "sync". I think it should always be "sync".